### PR TITLE
[2.0-preview-class] Add Travis support

### PR DESCRIPTION
**Pull request by abdonrd**
Also, I think can help @rictic to check the `polymer-linter`.

PS: Someone with permissions, needs to activate the repository in [Travis: /Polymer](https://travis-ci.org/Polymer).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: node_js
+dist: trusty
+sudo: required
+addons:
+  firefox: latest
+  apt:
```
Comment (stramel) [inline .travis.yml]: switch to `chrome: stable` instead?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      - google-chrome-stable
+node_js:
+  - node
+  - 6
+cache: yarn
+install:
+  - yarn global add polymer-cli
+  - polymer install
+before_script:
+  # - polymer lint
```
Comment (stramel) [inline .travis.yml]: I vote we add it now so the build fails and draws more attention to the lint warnings/errors.

Comment (abdonrd) [inline .travis.yml]: Reference to https://github.com/Polymer/shop/pull/122#discussion_r116103876

File: .travis.yml (hunk 1 above)
Comment (abdonrd) [inline .travis.yml]: I get the same addons from the other Polymer projects. 😕

**Comment by abdonrd** [comment]
Closed in favor to #122.

**Pull request closed without merging**